supervisor/tests/test_dispatchers.py

```diff
@@ -512,6 +512,76 @@ def test_syslog_logfile_deprecated(self):
             self._makeOne(process)
             self.assertEqual(len(w), 1)
 
+    def test_no_logfile_and_no_syslog(self):
+        from supervisor.datatypes import boolean, logfile_name
+        from supervisor.options import ProcessConfig, ServerOptions
+        options = ServerOptions() # need real options to get a real logger
+        options.loglevel = 20 # INFO, see loggers.py LevelsByName
```

Comment:
Existing tests in this file do things like `options.loglevel = loggers.LevelsByName.TRAC`.  Why is this different?

Reply:
This was an oversight. I'll change it.